satellite/metainfo/metainfo.go

```diff
@@ -579,3 +580,22 @@ func bytesToUUID(data []byte) (uuid.UUID, error) {
 
 	return id, nil
 }
+
+// ProjectInfo returns allowed ProjectInfo for the provided API key
+func (endpoint *Endpoint) ProjectInfo(ctx context.Context, req *pb.ProjectInfoRequest) (_ *pb.ProjectInfoResponse, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	keyInfo, err := endpoint.validateAuth(ctx, macaroon.Action{
+		Op:   macaroon.ActionProjectInfo,
+		Time: time.Now(),
+	})
+	if err != nil {
+		return nil, status.Errorf(codes.Unauthenticated, err.Error())
+	}
+
+	salt := sha256.Sum256([]byte(keyInfo.ProjectID.String()))
```

Comment:
`projectID.String()` returns UUID in this format `%x-%x-%x-%x-%x`. Do we want byte representation of this string or more `sha256.Sum256(keyInfo.ProjectID[:])`?

Reply:
oh i didn't realize uuid.UUID was a byte type. sure, that's better